run/README.namelist

```diff
@@ -656,7 +656,7 @@ Namelist variables for controlling the adaptive time step option:
                                      = 0, no surface-layer
                                      = 1, Revised MM5 Monin-Obukhov scheme (Jimenez, renamed in v3.6)
                                      = 2, Monin-Obukhov (Janjic) scheme
-                                     = 3, NCEP Global Forecast System scheme (NMM only)
+                                     = 3, NCEP Global Forecast System scheme (works for both ARW and NMM )
```

Comment:
Ming,
We only seem to comment when the scheme works for only a single dynamical core. In this mod and the next, remove the parenthetical comment.